benchmarks/CMakeLists.txt

```diff
@@ -32,7 +32,8 @@ function(ConfigureBench BENCH_NAME)
     ${BENCH_NAME}
     PROPERTIES POSITION_INDEPENDENT_CODE ON
                RUNTIME_OUTPUT_DIRECTORY "$<BUILD_INTERFACE:${RMM_BINARY_DIR}/gbenchmarks>"
-               CUDA_ARCHITECTURES "${CMAKE_CUDA_ARCHITECTURES}")
+               CUDA_ARCHITECTURES "${CMAKE_CUDA_ARCHITECTURES}"
```

Comment:
I don't see us installing benchmarks for other projects like cudf, so we shold drop this and stick with tests for now.

Reply:
@robertmaynard, for reference, the reason I included benchmarks is because I was trying to account for all of the Project Flash discrepancies in the `build.sh` conditional statement for the `librmm` `conda` recipe below. It seems that when Project Flash is enabled, benchmarks are built in addition to tests. So I thought it made sense to add the benchmark `install` rules in this PR so that we can include the benchmarks in the new `librmm_tests` package that we'll be creating in a subsequent PR. Do you agree with this approach? If not, are you suggesting that benchmarks go in their own conda package (i.e. something like `librmm_benchmarks`)? Personally I think it might be excessive to have two packages like that (i.e. `librmm_tests` and `librmm_benchmarks`). Let me know your thoughts.


https://github.com/rapidsai/rmm/blob/79f7e39ec89d4d9fdba46dd9f653cc7ccc523ef2/conda/recipes/librmm/build.sh#L4-L8